eml/components/courses/courseCard/DownloadCourseButton.jsx

```diff
@@ -3,6 +3,8 @@ import LottieView from 'lottie-react-native';
 import {Alert, TouchableWithoutFeedback} from 'react-native';
 import animationAsset from '../../../assets/animations/downloadAnimation.json';
 import PropTypes from 'prop-types';
+import * as StorageService from '../../../services/StorageService';
+import AsyncStorage from '@react-native-async-storage/async-storage';
```

Comment:
Quick question: Shouldn't we be using StorageService instead of AsyncStorage?

Reply:
We should. The function using the Asyncstorage should reside in Storage service. We should move it when I get to uni